core/trino-parser/src/main/java/io/trino/sql/parser/AstBuilder.java

```diff
@@ -731,7 +731,7 @@ public Node visitDropColumn(SqlBaseParser.DropColumnContext context)
     {
         return new DropColumn(getLocation(context),
                 getQualifiedName(context.tableName),
-                (Identifier) visit(context.column),
+                getQualifiedName(context.column),
```

Comment:
It seems the grammar always supported qualified name, but we expected to parse it as an identifier.

Add a prep commit with a case in TestSqlParser.testDropColumn that shows how the qualified "column name" used to be handled. This will help me review the consequences of the change here.

Reply:
Can you please add this prep commit with a test case?
it would go before any other changes.

Reply:
The commit is already extracted as https://github.com/trinodb/trino/pull/16002/commits/0f086432eea32431365e6a3f3d724ae2f6a67930 with test some test cases. Am I missing something?

Reply:
That commit contains tests but also functionality changes.
I don't know how the existing functionality changes, because it had no tests.
Please add the tests first, covering DROP COLUMN with qualified name. It was "always" supported by grammar. Did it fail later?

Reply:
I get it now. I thought you already saw the failure due to qualified name in #15975

Reply:
thanks for the pointer!